guides/doc-Provisioning_Guide/master.adoc

```diff
@@ -10,11 +10,47 @@ endif::[]
 
 include::common/attributes.adoc[]
 
-include::topics/Introduction.adoc[]
+[id="provisioning-introduction"]
+== Introduction
 
-include::topics/Provisioning_Contexts.adoc[]
+include::common/modules/con_provisioning-overview.adoc[leveloffset=+2]
+
+include::common/modules/con_provisioning-workflow.adoc[leveloffset=+2]
+
+include::common/modules/con_provisioning-contexts.adoc[leveloffset=+2]
+
+include::common/modules/con_provisioning-templates.adoc[leveloffset=+2]
```

Comment:
How about moving these two section 1.3. Provisioning Contexts and 1.4. Provisioning Templates to Chapter 2. Configuring Provisioning Resources?
1. By doing this we can move con_provisioning-contexts.adoc closer to  proc_setting-the-provisioning-context.adoc
2. Provisioning Contexts and Provisioning Templates are provisioning resources

Reply:
Now that you mention it, I wonder if con_provisioning-contexts.adoc is overkill and could just be a few lines at the start of Setting the Provisioning Context. I'm not sure whether the concept info on its own could potentially be reused in, say, the Planning guide. :shrug:

Reply:
Good point about the reuse. The current Planning guide has sections [3.1. Organizations 3.2. Locations](https://access.redhat.com/documentation/en-us/red_hat_satellite/6.6/html-single/planning_for_red_hat_satellite_6/index#chap-Red_Hat_Satellite-Architecture_Guide-Org_Loc_and_Life_Cycle_Environments) that can be substituted by this concept module in future.

Reply:
Cool, I'll keep the concept module, then!